csharp/extractor/Semmle.Extraction.CIL/Tuples.cs

```diff
@@ -203,14 +203,14 @@ internal static class Tuples
         internal static Tuple containerparent(Folder parent, IFileOrFolder child) =>
             new Tuple("containerparent", parent, child);
 
-        internal static Tuple files(File file, string fullName, string name, string extension) =>
-            new Tuple("files", file, fullName, name, extension, 0);
+        internal static Tuple files(File file, string fullName) =>
+            new Tuple("files", file, fullName);
```

Comment:
Can we get rid of `Semmle.Extraction.CIL.Tuples.files()` and `Semmle.Extraction.CIL.Tuples.folders()` methods altogether? The relations are shared between CIL and C#, so we could use a single implementation in `Semmle.Extraction.Tuples`.